### ISSUE TITLE
ADD: view code

### ISSUE THREAD

**Issue by MacroMachines** [beautypi/shadertoy-iOS-v2#18]
would be very useful to be able to at the very least view the code, I do most of my shader development using Shaderific, Codea, and Verto Studio on iOS and its impossible to copy or edit code from the ShaderToy site on iOS.

**Comment by reindernijhoff** [comment]
Yes, I will add this functionality in a future update of the app.
